README.md

```diff
@@ -84,52 +90,68 @@ See [requirements.txt](requirements.txt) for dependencies.
 # Installation
 
 ## Docker
+
 ### Clone the repository
-`git clone git@github.com:dolevf/Damn-Vulnerable-GraphQL-Application.git && cd Damn-Vulnerable-GraphQL-Application`
+
+`git clone https://github.com/dolevf/Damn-Vulnerable-GraphQL-Application.git dvga && dvga`
```

Comment:
I rather not change the name of the directory here

Reply:
More importantly `cd` was missing.